daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -453,6 +456,7 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
     banner("""|Usage: daffodil unparse (-s <schema> [-r <root>] | -P <parser>)
               |                        [-c <file>] [-D<variable>=<value>...] [-I <infoset_type>]
               |                        [-o <output>] [--stream] [-T<tunable>=<value>...] [-V <mode>]
+              |                        [--trace | --debug <file>]
               |                        [infile]
```

Comment:
These usage strings are starting to get a bit verbose. I'm wondering they are getting too complicated to be useful and if we should simplify the usage string to just something like this:

```
Usage: daffodil unparse (-s <schema> | -P <parser>) [UNPARSE_OPTS] [--] [infile]

Unparse Options:
  -- scallop generated stuff--
```

So it shows only the required and commonly used options (-s and -P) and trailing args, and anything else is described in more detail below that. We would probably want to also modify some descriptions to mention restrictions. For example, '--root' would need to say that it's only valid with the --schema option. But that and trace/debug mutual exclusion are the only restrictions I think.

It also makes the `[--]` stand out a bit more so might make it's availability more clear. And it means we don't have to update this usage and make it even more complicated every time we add a new option.